app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/MainFragment.kt

```diff
@@ -113,7 +113,7 @@ class MainFragment : Fragment() {
              */
             for(x in 0 until 5) {
                 try{
-                    xAxisLabels.add(caseData.country_name[x])
+                    xAxisLabels.add(caseData.countryName[x])
```

Comment:
Changed to better fit Kotlin naming conventions https://kotlinlang.org/docs/reference/coding-conventions.html